### PR TITLE
Adding lstrip & rstrip option to write method.

### PR DESCRIPTION
**Pull request by makkus**
Hi, as mentioned in issue #16 , the write method is fairly strict and doesn't allow leading whitespaces. This commit adds lstrip & rstrip options to give more flexibility.

I didn't add tests yet, as I don't have any experience with pytest fixtures, so couldn't easily figure out exactly how to best do this for yaspin. I might give it another shot in a few days if I find some time.

### CODE REVIEW

File: yaspin/yaspin.py [hunk 1]
```diff
@@ -183,14 +183,32 @@ def show(self):
             sys.stdout.write("\r")
             self._clear_line()
 
-    def write(self, text):
-        """Write text in the terminal without breaking the spinner."""
+    def write(self, text, lstrip=True, rstrip=True):
```
Comment (pavdmyt) [inline yaspin/yaspin.py]: After some rethinking I'm coming to conclusion that `write(self, text, strip=True)` would be a better API. `strip=True` will allow to save a default `write` behavior and `strip=False` allows user to strip a text in desired way before calling the `write` method. I think such approach would be more simple and flexible.

What do you think?

Comment (makkus) [reply yaspin/yaspin.py]: Yep, true.

**Comment by pavdmyt** [comment]
Hi @makkus 

Thanks for your PR!
I'll cover the unittests part.

**Comment by pavdmyt** [comment]
@makkus I've checked the source code and ran related tests again. It seems to me that there were no specific purpose for text stripping behavior. So, I just removed `.strip()` call. Now user can strip text in any desired way. Regarding backward compatibility, I think such change is OK, since we're still in beta and API is not yet stable.

Are you Ok with such solution to issue #16 ? If yes, I'll merge this PR and push new version to pip.

**Comment by makkus** [comment]
Yes, I'd be more than happy with this. Thanks a lot!

**Pull request merged**